### ISSUE TITLE
Both Quick and Full Library Sync fail w/ upgrade to 1.6.6

### ISSUE THREAD

**Issue by richardpickett** [openaudible/openaudible#309]
Ubuntu 19.10 w/ OA 1.6.6

Expected action: Either Quick Library Sync or Full Library Sync would sync the latest purchases

Actual action: Both Quick Library Sync and Full Library Sync produce the dialog:

> Unable to connect right now.
> 
> Try logging on to Audible from this web page and try again.
> 

However, I'm clearly logged in as the Library List is displayed in the browser (and I had already entered email/pass to login).

Steps to reproduce:

1. Start OpenAudible.
2. Controls -> Connect to Audible
3. Enter email/pass, get redirected to library page in audible
4. Controls -> Quick Library Sync  OR  Controls -> Full Library Sync
5. "Scanning your library to get your list of books" dialog displays
6. Either of these now happen:
6.a. It throws the error indicated above ("Unable to connect") OR
6.b. Occasionally it returns quickly with no error, but the Open Audible book list is not updated.

![screenshot](https://user-images.githubusercontent.com/1056873/78503161-9035d400-772a-11ea-9d8b-736a14cbdd56.png)

**Comment by richardpickett** [comment]
I walked back previous versions to see which version I could get to work.

1.6.5 exhibits the same problem.

1.6.4 works, with the caveat I had to make repeated requests to "connect to audible" even though I had already logged in and the browser was sitting on the library page.  Finally OA realized it was logged in and everything else worked just fine (including downloading a book and converting to mp3).
